.ci/go-test.sh

```diff
@@ -27,24 +29,36 @@ function run_as_user
 
 function test_html_coverage
 {
+	html_report="coverage.html"
+
 	test_coverage
-	go tool cover -html=profile.cov -o coverage.html
-	rm -f profile.cov
+
+	go tool cover -html="${cov_file}" -o "${html_report}"
+	rm -f "${cov_file}"
+
+	run_as_user "current" chmod 644 "${html_report}"
 }
 
 function test_coverage
 {
-	cov_file="profile.cov"
-	tmp_cov_file="profile_tmp.cov"
 
 	echo "mode: atomic" > "$cov_file"
 
 	for pkg in $test_packages; do
 
-		# Run the unit-tests *twice* (since some must run as root and
-		# others must run as non-root), combining the resulting test
-		# coverage files.
-		for user in non-root root; do
+		if [ $(id -u) -eq 0 ]
```

Comment:
I think you should move this ```if ... else ``` condition before ```for pkg in $test_packages; do```

Reply:
good catch! moving...

Reply:
Branch updated.